### PR TITLE
Fix dup-selected for pandas loaders and add test

### PR DESCRIPTION
**Pull request by jasonwirth**
- [ Y ] If contributing a core loader, [the loader checklist](https://visidata.org/docs/contributing#loader) was referenced.
- [ ] If registering an external plugin, [the plugin checklist](https://visidata.org/docs/contributing#plugins) was referenced.

Fix for [#1315 ](https://github.com/saulpw/visidata/issues/1315)

I added a sample .vd file as well as a sample .parquet file in `/sample_data`

### CODE REVIEW

**Comment by anjakefala** [comment]
The tests are failing because we are missing a golden result for `sample_parquet.vd`!

Could you please `vd -p tests/sample_parquet.vd`, check if the output looks as expected, and then `Ctrl+S` *tests/golden/sample_parquet.tsv*, and commit the saved file?

File: tests/select_parquet.vd [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sheet	col	row	longname	input	keystrokes	comment
```
Comment (anjakefala) [inline tests/select_parquet.vd]: I really appreciate you taking the time to add a test for the bug fix!

**Comment by jasonwirth** [comment]
I added the golden tests file but the CI fails with this [error](https://app.circleci.com/pipelines/github/saulpw/visidata/1162/workflows/812dbc16-f9be-451d-b82b-6f310f11e6cc/jobs/3969?invite=true#step-110-1323) which is really about needing a parquet library:


```
 - Missing optional dependency 'fastparquet'. fastparquet is required for parquet support. Use pip or conda to install fastparquet.
```

I can reproduce the issue without using parquet, it's just that I discovered the bug while reading parquet files. Since parquet isn't natively supported by VD (unlike csv) it'll fallback to pandas `read_parquet` method. I have to manually specify the loader to make it error again.

Let me create another tests uisng a .tsv.

**Review by anjakefala — APPROVED** [review]

**Comment by anjakefala** [comment]
Thanks so much @jasonwirth! For the fix, and the tests. =)

**Pull request merged**